overlord/hookstate/ctlcmd/ctlcmd.go

```diff
@@ -32,10 +32,19 @@ import (
 	"github.com/snapcore/snapd/strutil"
 )
 
+func createMissingContextError(subcommand string) error {
+	return fmt.Errorf(`Cannot invoke snapctl operation commands (here %q) from outside of a snap`, subcommand)
```

Comment:
AFAICT errors in go lang shouldn't be capitalized (because they can be appended/wrapped in another error message and it then looks weird); I'm slightly surprised it's not caught by our static checks, I could swear I saw an error about such issue before...

Reply:
It's because I used a cyrillic "С" instead ;-)

Fixed now.

Reply:
Thanks